infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -550,10 +598,39 @@ public static LogUnitServerConfig parse(Map<String, Object> opts) {
             return LogUnitServerConfig.builder()
                     .cacheSizeHeapRatio(cacheSizeHeapRatio)
                     .maxCacheSize((long) (Runtime.getRuntime().maxMemory() * cacheSizeHeapRatio))
-                    .memoryMode(Boolean.valueOf(opts.get("--memory").toString()))
+                    .memoryMode(Boolean.parseBoolean(opts.get("--memory").toString()))
                     .noVerify((Boolean) opts.get("--no-verify"))
                     .noSync((Boolean) opts.get("--no-sync"))
                     .build();
         }
     }
+
+    /**
+     * Utility class used by the LogUnit server.
+     */
+    public static class LogUnitServerInitializer {
```

Comment:
I guess this class is for testing purpose only? If true, please mention this in the Javadoc.

Reply:
Have a look at the Javadoc of `SequencerInitializer` (previously `SequencerFactoryHelper`) in `SequencerServer.java`. You can copy the relevant info if required.

Reply:
Expanded the Javadoc to include this information.